### ISSUE TITLE
Automated Release Process for SGrPython

### ISSUE THREAD

**Issue by robin-schoch** [SmartGridready/SGrPython#28]
### Issue Description
Right now new release of the SGrPython library are build und pushed locally by a developer. This leads to unwanted dependencies on some developers.

### Proposed Solution
Implementing a CI/CD pipeline would automate and streamline the release process:
- Introduce Semantic Versioning (Semver) to SGrPython.
- Add Github Actions
- On merge into Develop
  - Build and create a Python module with Semver-beta
  - Push package to PyPi
- On merge into Master
  - Build and create a Python module with Semver
  - Push package to PyPi
  - Increment Semver

**Comment by mkrebs81** [comment]
Suggestion for change:

- publish snapshot (beta) after commit to master
- publish release only after creating tag on master
